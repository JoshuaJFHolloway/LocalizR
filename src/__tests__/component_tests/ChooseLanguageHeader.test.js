import React from 'react';
import { shallow } from 'enzyme';
import ChooseLanguageHeader from '../../components/ChooseLanguageHeader';

describe('ChooseLanguageHeader', () => {
  let chooselanguageheader = shallow(<ChooseLanguageHeader />);

  it('renders correctly', () => {
    expect(chooselanguageheader).toMatchSnapshot();
  });
});
