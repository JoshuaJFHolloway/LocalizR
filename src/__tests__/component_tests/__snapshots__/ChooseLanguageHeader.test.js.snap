// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ChooseLanguageHeader renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <ChooseLanguageHeader />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": " Hi __name+here__, choose your language ",
    },
    "ref": null,
    "rendered": " Hi __name+here__, choose your language ",
    "type": "h1",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": " Hi __name+here__, choose your language ",
      },
      "ref": null,
      "rendered": " Hi __name+here__, choose your language ",
      "type": "h1",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
    "disableLifecycleMethods": true,
  },
}
`;
